### PR TITLE
Pep8 compliance, add coverage badge

### PR DESCRIPTION
**Pull request by lsav**
Giant pile of whitespace fixes + 96% code coverage yay

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    W504  # linebreak after binary operator
+max-line-length = 100
+exclude =
+    .git,
+    __pycache__,
+    docs,
+    ephemeris,
+    kenv,
+    scripts,
+    test
```
Comment (ZeyadTamimi) [inline .flake8]: NEW LINE AGHGHGHGHGHGHG

File: kaos/tuples.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 from collections import namedtuple
 
+# pylint: disable=pointless-string-statement
 """Representation a 3D cartesian vector"""
```
Comment (ZeyadTamimi) [inline kaos/tuples.py]: Why is this a string statement?

File: .coveragerc [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+[report]
+exclude_lines =
+    if self.debug:
+    pragma: no cover
+    raise NotImplementedError
+    if __name__ == .__main__.:
+ignore_errors = True
+omit =
+    test/*
```
Comment (ZeyadTamimi) [inline .coveragerc]: NEWE LINE AGHGAHGHGHGHGHGHGH

File: kaos/tuples.py (hunk 2 above)
Comment (lsav) [reply kaos/tuples.py]: That was meant to be a docstring. Upon further research, apparently this won't work for Sphinx. Pushing an update with fix.

ref: https://stackoverflow.com/a/28568351

File: .coveragerc (hunk 3 above)
Comment (lsav) [reply .coveragerc]: 😩

File: kaos/tuples.py (hunk 2 above)
Comment (ZeyadTamimi) [reply kaos/tuples.py]: Tyty

**Comment by lsav** [comment]
afaik the codecov thing will fix itself once Travis runs on master.

**Pull request merged**